.github/workflows/ci-base.yml

```diff
@@ -0,0 +1,44 @@
+name: PR Releases
+
+on:
+  workflow_call:
+    inputs:
+      os:
+        description: 'Operating System to run the workflow on'
+        required: true
+        type: string
+      node-version:
+        description: 'Node.js version to use'
+        required: true
+        type: string
+      publish-extra-args:
```

Comment:
I would avoid this. I'm not sure we need it anywhere. I feel like it would be better to expose specific flags through options to avoid future incompatible setups being created on the consumer side.

Reply:
Removed.